REQUIREMENTS.md

```diff
@@ -76,7 +76,7 @@ generated requirements.
 
 Your second command is actually more like:
 ```
-$ pip-compile -o requirements.txt requirements.txt requirements.in
+$ pip-compile -o requirements.txt requirements.in
```

Comment:
The proposed change is incorrect. If this section is unclear, we should change it. In the context of lines 57- 82, I think line 79 makes sense.